### ISSUE TITLE
Truncate long strings etc?

### ISSUE THREAD

**Issue by indigoviolet** [samuelcolvin/python-devtools#68]
I noticed in #16 that you intended to strip the middle of massive outputs, but I can't seem to get this to happen (for very long strings). Is it possible to configure this somehow?

**Comment by samuelcolvin** [comment]
i think I never did it, mostly I find it useful to show the whole of even very long strings.

Have to accept a PR, but it would need to be configurable, and default to off.

**Comment by alecgerona** [comment]
This would also be useful for long dictionaries or lists.

**Comment by samuelcolvin** [comment]
This would **sometimes** be useful for long dictionaries or lists.

In other cases it would be extremely annoying, this would need to be an opt-in feature.

**Comment by sshishov** [comment]
Agree, when usually people are debugging something, they want to see the full object, not just some part of it as the valuable piece of code can be just truncated most of the time 🤣
